docs/_posts/examples/3400-01-31-data-join.html

```diff
@@ -0,0 +1,73 @@
+--- 
+layout: example 
+category: example 
+title: Join client data to vector tiles 
+description: Use data-driven styles to join client data to vector tiles
+tags: 
+  - sources 
+---
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map'
+    style: 'mapbox://styles/mapbox/light-v9',
+    center: [-99.9, 41.5],
+    zoom: 1,
+});
+
+//Get the client data to join 
+var data = {
+    "01": 10,
+    "02": 12,
+    "06": 20,
+    "10": 15,
+    "15": 10,
+    "20": 20,
+    "25": 1,
+    "30": 8,
+    "35": 4,
+    "40": 18,
+    "45": 11,
+    "50": 3
+}
+
+// Get the vector data to join
+var mapid = 'rsbaumann.8h761oct' 
+var layername = 'states'       
+var vt_match_prop = 'STATE_ID' 
+
+map.on('load', function() {
+
+    // Add source for state polygons hosted on Mapbox
+    map.addSource('states', {
+        type: 'vector',
+        url: 'mapbox://' + mapid
+    });
+
+    // First value is the default, used where the is no data
+    var stops = [["0", "rgba(0,0,0,0)"]];
+
+    // Calculate color based on the data value
+    Object.keys(data).forEach(function(key) {
+        let val = parseInt(data[key])
```

Comment:
We should either use `let`/`const` everywhere, or `var` everywhere — let's not mix the styles. I'd suggest `var` to keep things in line with other examples.